### DISCUSSION TITLE
how to change password casaOS

### DISCUSSION THREAD

**Discussion by rsbyxx** [IceWhaleTech/CasaOS#1931] [General]
### **Hello, I want to ask how to change our Casa OS admin password?**
I want to change my password but I can't find a menu to change the password
my os Armbian linux
device RASPI 4

**Comment by dragonfire1119** [comment]
You can update your password by navigating to the UI and selecting “Account” from the top left menu. Click the pencil icon next to the password field. Additionally, you can follow this video tutorial for further guidance:

https://youtu.be/m0Rsu9h7Z44

**Comment by rsbyxx** [comment]
### solved 

 run this script in your root
bash -c "$(wget -qLO - https://raw.githubusercontent.com/bigbeartechworld/big-bear-scripts/master/reset-password-for-casaos/run.sh)"

u can reset passwor casa os
thanks

**Reply by Kisaragilice** [reply]
thank you sir
